content/3.0templating.md

```diff
@@ -31,11 +31,11 @@ parameter while executing the template, if you are wondering how we'd do that, t
 		}
 
 These are the three parts of using templates, first you need to create types like we have created the `Context` type, then we need to read the template, 
-then we need to use the components of that type in our template file. So what remains now is passing an object of that type while template execution.
+then we need to use the components of that type in our template file. So what remains now is passing an object of that type during template execution.
 
 Reading template:
 
-		templates, err = template.ParseFiles(allFiles...)
+		templates, err = template.ParseGlob(allFiles...)
```

Comment:
@benkasminbullock 
ParseGlob is equivalent to calling ParseFiles with the list of files matched by the pattern. 
So shouldn't we be doing `template.ParseGlob("*.html")`?